src/parameterBuilder.js

```diff
@@ -48,14 +70,13 @@ module.exports = function( request, params ) {
 		if ( Array.isArray( param.val ) ) {
```

Comment:
You could eliminate some nesting by inverting this condition.

Reply:
Swapped this up a bit. What do you think?